generators/client/templates/angular/src/main/webapp/app/app-routing.module.ts.ejs

```diff
@@ -53,7 +53,7 @@ const LAYOUT_ROUTES = [navbarRoute, ...errorRoute];
         <%_ } _%>
         ...LAYOUT_ROUTES,
       ],
-      { enableTracing: DEBUG_INFO_ENABLED }
+      { enableTracing: DEBUG_INFO_ENABLED, relativeLinkResolution: 'corrected' }
```

Comment:
the default is 'corrected' in angular 11, should we just take it out in this case ?

Reply:
We need to check, does we have code which relies on `relativeLinkResolution`, if yes then migrate that code and then remove this.